scripts/xtensa-build-zephyr.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+# stop on most errors
+set -e
+
+SUPPORTED_PLATFORMS=(apl cnl icl tgl-h)
+BUILD_PATH=zephyr
+BUILD_ROM=no
+BUILD_JOBS=$(nproc --all)
+RIMAGE_KEY=modules/audio/sof/keys/otc_private_key.pem
+PLATFORMS=()
+
+PATH=$pwd/local/bin:$PATH
+
+pwd=$(pwd)
+
+die()
+{
+	>&2 printf '%s ERROR: ' "$0"
+	# We want die() to be usable exactly like printf
+	# shellcheck disable=SC2059
+	>&2 printf "$@"
+	exit 1
+}
+
+print_usage()
+{
+    cat <<EOF
+Re-configures and re-builds SOF with Zephyr using a pre-installed Zephyr toolchain and
+platform's _defconfig file.
+
+usage: $0 [options] platform(s)
+
+       -a Build all platforms
+       -j n Set number of make build jobs. Jobs=#cores by default.
+           Infinte when not specified.
+       -k Path a non-default rimage signing key.
+       -p Build path.
```

Comment:
Is this path to build directory ? Also what about path to Zephyr ?

Reply:
@lgirdwood The comment might be not very clear, this is path to Zephyr, its contents (reduced) will be
> zephyr-root/
>├── bootloader
>├── build-apl
>├── build-rimage
>├── modules
>├── tools
>└── zephyr

where `build-apl` and `build-rimage` are created by this script for building, the rest is created by west